taskcluster/ci/beetmover/kind.yml

```diff
@@ -24,6 +24,7 @@ only-for-build-types:
 job-template:
     attributes:
         artifact_map: taskcluster/fenix_taskgraph/manifests/fenix_candidates.yml
+        shipping_phase: ship
```

Comment:
shouldn't pushing to candidates be part of promote?

Reply:
I was confused by the name of the `artifact_map` too! As far as I can see, the only beetmover task publishes APKs to the `release` folder and not the `candidates` one. I'm okay renaming the yaml file, if you think that's the right thing to do.

Reply:
Seems worth renaming the map for clarity (possibly in another PR).  And then as a followup, I think we should consider actually pushing to candidates in promote and adding a push-to-releases task in the ship phase.

Reply:
I definitely agree, I was thinking the same 👍

Reply:
Done as part of this PR.